lib/blacklight_range_limit/view_helper_override.rb

```diff
@@ -89,8 +89,10 @@ def range_config(solr_field)
     def range_params(my_params = params)
       return {} unless my_params[:range].is_a?(ActionController::Parameters) || my_params[:range].is_a?(Hash)
 
+      facet_field_keys = blacklight_config.facet_fields.keys
       my_params[:range].select do |_solr_field, range_options|
```

Comment:
By convention, now that `_solr_field` is used, we should drop the `_`. I also don't think we need to memoize the list of keys, right?

```suggestion
      my_params[:range].select do |solr_field, range_options|
        next unless range_options && blacklight_config.facet_fields[solr_field].present?
```

And, fwiw, this should be the behavior once we get range limit using components too.

Reply:
Yeah, that makes total sense.  Thanks.